src/math/warpExposure.cc

```diff
@@ -595,7 +596,8 @@ int afwMath::warpImage(
     typename DestImageT::SinglePixel padValue
 ) {
     afwGeom::Point2D const destXY0(destImage.getXY0());
-    afwMath::detail::WcsPositionFunctor const computeSrcPos(destXY0, destWcs, srcWcs);
+    afwImage::XYTransformFromWcsPair xyTransform{destWcs.clone(), srcWcs.clone()};
```

Comment:
I agree that it is disappointing, but I want to push back here. Warping is already slow, and I doubt that the expense of cloning the WCS is a significant factor. I propose we leave it until we get warping fast enough that the slowdown is a significant concern.